src/modules/Hosts/Hosts/Views/MainPage.xaml

```diff
@@ -37,25 +37,23 @@
             TrueValue="Collapsed" />
     </Page.Resources>
 
-    <Grid>
+    <Grid
+        Margin="16"
+        RowSpacing="8">
         <Grid.RowDefinitions>
-            <RowDefinition Height="64" />
-            <RowDefinition Height="*" />
             <RowDefinition Height="Auto" />
+            <RowDefinition Height="*" />
         </Grid.RowDefinitions>
 
         <Grid>
             <Button
                 x:Uid="AddEntryBtn"
-                Height="36"
-                Margin="18,0,0,0"
```

Comment:
The 18 margin was added to fix this effect where the top buttons looks wider than the hosts list (same for setting button at right):

![image](https://github.com/microsoft/PowerToys/assets/25966642/858a9beb-32f3-455b-a682-758c5265d3f6)

Reply:
I think this is an optical illusion. Using ScreenRuler I see that on the left: visual margins are 86 and 76; where on the right both are 76.

Reply:
You are right. It's move visible with the dark theme and it's related to the ListView border.

![2](https://github.com/microsoft/PowerToys/assets/25966642/f7ac7c33-e519-474e-8a54-b7d2b29c1c23)

![1](https://github.com/microsoft/PowerToys/assets/25966642/5faa0fa3-1291-45af-8ff0-c6325730caa8)

I really don't like the effect with dark theme. TBH I will be fine with an inconsistent margin.
Any thoughts on this @niels9001?